### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by PyCore23**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,22 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+    }
```
Comment (pavlejviki) [inline app/main.py:8]:
```suggestion
    return {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"],
        "name": error["code"],
        "source": "flake8"
    }
```

Comment (pavlejviki) [reply app/main.py:8]: follow this formatting in the second function as well

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+            "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        format_single_linter_file(file_path, error)
+        if error is not None else []
```
Comment (pavlejviki) [inline app/main.py]: if, else is redundant

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**